src/Firehose.Web/Authors/AlexandrSorokoletov.cs

```diff
@@ -0,0 +1,24 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.ServiceModel.Syndication;
+using Firehose.Web.Infrastructure;
+
+namespace Firehose.Web
+{
+    public class AlexandrSorokoletov : IAmAMicrosoftMVP
+    {
+        public string FirstName => "Alexandr";
+        public string LastName => "Sorokoletov";
+        public string ShortBioOrTagLine => "I plan, design and develop mobile products to achieve measurable business results";
```

Comment:
PlanetXamarin isn't LinkedIn or your resume. Have a look at the tone/voice of some of the other author's taglines.  Keep it human, not business. i.e. "Tomasz Cielecki is a Xamarin MVP who loves long walks on the beach, yelling at the screen. More importantly writes code."